packages/libraries/federation-link-utils/src/index.ts

```diff
@@ -46,12 +46,6 @@ export function extractLinkImplementations(typeDefs: DocumentNode): {
 
   return {
     resolveImportName: (identity, name) => {
-      if (!supportsFederationV2) {
-        // Identities dont matter for Federation v1. There are no links to reference.
-        // So return the name without the identity's namespace
-        return name.startsWith('@') ? name.substring(1) : name;
```

Comment:
Im hitting issues with this logic. It isnt completely accurate...
To know for sure I need to add a check for the `@core` directive also.
WIP